src/queue/Playlist.hxx

```diff
@@ -102,6 +102,9 @@ struct playlist {
 	[[gnu::pure]]
 	int GetNextPosition() const noexcept;
 
+	const std::string getLastLoadedPlaylist() const noexcept;
```

Comment:
Bad method name, but the worst thing is that this allocates a copy of the string on the heap.

Reply:
Renamed to starting uppercase and converted to std::string_view. I hope this is correct!